src/config/generate_function_list.sh

```diff
@@ -13,4 +13,6 @@
 shopt -s nullglob
 
 # Use awk to avoid issues with spacing
-nm -P *.o *.obj | awk '$2 == "T" {print $1}' | sed -e 's/^_//' -e 's/_$//'
+# Demangle any c++ name mangling and filter _device_stub_ prefixes.
+nm -P *.o *.obj | awk '$2 == "T" {print $1}' | c++filt | sed -e 's/(.*$//' -e 's/^__device_stub__//' -e 's/^_//' -e 's/_$//'
```

Comment:
Have you looked into the portability of c++flt?

Reply:
Yes, I have thought a bit about this. From what I can tell, it is supported in binutils, macOS, most BSD systems, MinGW/Cygwin and so should be fairly portable. Ideally we would use the demangle option the "-C" for nm, since it is built into the function, but unfortunately work well. There is also --demangle for nm on GNU/linux systems, but c++filt is more portable.